vendor/tektelic/t00059xx-codec.yaml

```diff
@@ -0,0 +1,2 @@
+uplinkDecoder:
+  fileName: decoder_agriculture_sensor.js
```

Comment:
Hi @proffalken can you provide an example?

Reply:
@Jaime-Trinidad - sure, that's the one bit that I'm missing from the docs.

Should the example contain the raw payload, example output, both, or something else completely?

Reply:
@proffalken yes payload and expected decode uplink data according to your file, you can follow the example-codec.yalm it would be great if we can have it. Thank you

Reply:
OK, I'll find some time to do that in the next few days.